scalafix-internal/rules/src/main/scala-2/fix/UseLiteralsSyntax.scala

```diff
@@ -33,7 +33,7 @@ class UseLiteralsSyntax extends SemanticRule("Http4sUseLiteralsSyntax") {
             List(lit @ Lit.String(_))
           ) =>
         Patch.replaceTree(t, s"path$lit") + importLiteralsIfNeeded
-      case t if t.syntax == """Uri.Path.unsafeFromString("foo/bar")""" => show(t)
+      case t if t.text == """Uri.Path.unsafeFromString("foo/bar")""" => show(t)
```

Comment:
That change comes from https://github.com/scalameta/scalameta/releases/tag/v4.10.0
It echoes with https://github.com/scalameta/scalameta/pull/3864

Reply:
By the way, that line seems to be for debugging purposes. I'm not sure we should keep it.